app/uk/gov/hmrc/bindingtariffclassification/controllers/CaseController.scala

```diff
@@ -28,12 +28,15 @@ import scala.concurrent.Future
 @Singleton
 class CaseController @Inject()(caseService: CaseService, caseParamsMapper: CaseParamsMapper) extends CommonController {
 
-  import JsonFormatters.formatCase
-  import JsonFormatters.formatStatus
+  import JsonFormatters.{formatCase, formatStatus}
 
   def create: Action[JsValue] = Action.async(parse.json) { implicit request =>
     withJsonBody[Case] { caseRequest: Case =>
-      caseService.insert(caseRequest) map { c => Created(Json.toJson(c)) }
+      if(caseRequest.reference != null) {
+        Future.successful(BadRequest("Case reference is not permitted for a Create Request"))
```

Comment:
Thinking the request is well-formed. Maybe 422  could be more precise for this one. 400 is generic and works but is just an idea. 
extracted from web: HTTP status code 422 feels like a much more appropriate response for situations where the data is understood, but is still not valid. I think I'll start using this going forward

Reply:
Potentially, although I think thats for when you send in say a String when it should be a number

Reply:
I think both reasons (numeric or not permitted) are valid for this scenario...

Reply:
changed